lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -148,6 +148,183 @@ module Resources
       }
     }.freeze
 
+    R4_MEDICATION_REQUEST_PATIENT_ENTRY ||= {
+      'resourceType': 'MedicationRequest',
+      'id': '262448077',
+      'meta': {
+        'versionId': '2',
+        'lastUpdated': '2020-12-09T05:59:32.000Z'
+      },
+      'text': {
+        'status': 'generated',
+        'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Medication Request</b></p>/
+        <p><b>Status</b>: Active</p><p><b>Intent</b>: Order</p>
+        <p><b>Medication</b>: influenza virus vaccine, inactivated (influenza virus vaccine intramuscular solution)</p>/
+        <p><b>Dosage Instructions</b>: 0.5 mL, Subcutaneous, Once, 4, PRN: agitation</p>/
+        <p><b>Patient</b>: MARSTON, JACK</p>/
+        <p><b>Authored On</b>: Sep 16, 2020 10:16 A.M. CDT</p></div>'
+      },
+      'status': 'active',
+      'intent': 'order',
+      'category': [
+        {
+          'coding': [
+            {
+              'system': 'http://terminology.hl7.org/CodeSystem/medicationrequest-category',
+              'code': 'inpatient',
+              'display': 'Inpatient',
+              'userSelected': false
+            }
+          ]
+        }
+      ],
+      'priority': 'routine',
+      'reportedBoolean': false,
+      'medicationCodeableConcept': {
+        'coding': [
+          {
+            'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/synonym',
+            'code': '2760938',
+            'display': 'influenza virus vaccine intramuscular solution'
+          }
+        ],
+        'text': 'influenza virus vaccine, inactivated (influenza virus vaccine intramuscular solution)'
+      },
+      'subject': {
+        'reference': 'Patient/166045489',
+        'display': 'MARSTON, JACK'
+      },
+      'encounter': {
+        'reference': 'Encounter/49785642'
+      },
+      'authoredOn': '2020-09-16T10:16:07.000-05:00',
+      'requester': {
+        'reference': 'Practitioner/605926',
+        'display': 'Carter, Kristen Cerner'
+      },
+      'courseOfTherapyType': {
+        'coding': [
+          {
+            'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4009',
```

Comment:
Should be 'system": 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4009'

Reply:
comment addressed